scripts/in_container/check_environment.sh

```diff
@@ -107,8 +107,12 @@ function startairflow_if_requested() {
         . "$( dirname "${BASH_SOURCE[0]}" )/configure_environment.sh"
 
         # initialize db and create the admin user if it's a new run
-        airflow db init
-        airflow users create -u admin -p admin -f Thor -l Adminstra -r Admin -e dummy@dummy.email
+        if [[ ${RUN_AIRFLOW_1_10} == "true" ]]; then
+            airflow initdb
+        else
+            airflow db init
+            airflow users create -u admin -p admin -f Thor -l Adminstra -r Admin -e dummy@dummy.email
```

Comment:
```suggestion
            airflow users create -u admin -p admin -f Thor -l Adminstra -r Admin -e dummy@dummy.email || true
```
This will help in case of no RBAC.

Reply:
Actually that line is only for version 2 onward, but it gave me an idea to include `create_user` also for the RBAC setup. See my next commit.

Reply:
@potiuk there won't be any RBAC option in 2.0 anymore, right?

Reply:
Ther won't be indeed. But we are still able to install 1.10 in the ./breeze from 2.0 (for convenience) via `--install-airflow-version` . So we should add this option just for that case. I am going to do it shortly.